src/helpers/LevelJson.h

```diff
@@ -0,0 +1,209 @@
+// bbeck13
+
+#ifndef LEVEL_JSON_H_
+#define LEVEL_JSON_H_
+#include <vector>
+#include <iostream>
+#include <glm/ext.hpp>
+
+#include "MovingPlatform.h"
+#include "DroppingPlatform.h"
+#include "Platform.h"
+#include "json.hpp"
+
+namespace glm {
+void to_json(nlohmann::json& j, const glm::vec3& vec3) {
+  j = nlohmann::json{{"x", vec3.x}, {"y", vec3.y}, {"z", vec3.z}};
+}
+
+void from_json(const nlohmann::json& j, glm::vec3& vec3) {
+  vec3.x = j["x"].get<float>();
+  vec3.y = j["y"].get<float>();
+  vec3.z = j["z"].get<float>();
+}
+}
+
+namespace std {
```

Comment:
That's how the json parser we're using works, the to/from json methods have to be declared in the namespace they apply to